### PR TITLE
Back Button Work on the pet details activity

### PR DESCRIPTION
**Pull request by cbaja**
Back Button Issue fix on the pet details activity

### CODE REVIEW

**Comment by cbaja** [comment]
@scottrichards @smacgregor

**Comment by smacgregor** [comment]
hmm this branch has merge conflicts with master and can't be merged.

**Comment by smacgregor** [comment]
Try merging master into your branch?

File: app/src/main/java/com/codepath/apps/critterfinder/activities/PetDetailsActivity.java [hunk 1]
```diff
@@ -50,4 +53,15 @@ private void setupPetDetails(PetModel pet) {
...
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        if (item.getItemId() == android.R.id.home) {
+            // for now the back button will save the results
+            finish();
+            return true;
+        } else {
+            return super.onOptionsItemSelected(item);
+        }
+    }
```
Comment (smacgregor) [inline app/src/main/java/com/codepath/apps/critterfinder/activities/PetDetailsActivity.java:66]: this looks great! That's the piece we were missing before. Nice.

**Comment by smacgregor** [comment]
I resolved the merge conflicts and fixed the issue with the on click listener being static.

Verified the back button works! Nicely done.

**Pull request merged**